root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js

```diff
@@ -165,6 +166,15 @@ function Fx_OnSubmit() {
         else if (navigator.appVersion.indexOf("MSIE 8.0") != -1) {
             // IE8.0では完全に有効
         }
+        else if (navigator.appVersion.indexOf("MSIE 9.0") != -1) {
+            // IE9.0で問題の報告を受けていません。 
+        }
+        else if (navigator.appVersion.indexOf("MSIE 10.0") != -1) {
+            // IE10.0で問題の報告を受けていません。 
+        }
+        else if (navigator.appVersion.indexOf("Trident/7") != -1) {
+            // IE11.0で問題が合った場合、報告をお願いします。
+        } 
```

Comment:
I will register this improvement to "issue" as backlog.
This improvement is to add the code that is to determine browser type rather than the version.